assets/src/components/modal.tsx

```diff
@@ -23,6 +25,22 @@ const Modal = (): ReactElement | null => {
     return <InactiveNotificationModal notification={selectedNotification} />
   }
 
+  if (
+    selectedNotification &&
+    selectedNotification.reason == "chelsea_st_bridge_raised"
+  ) {
+    return (
+      <ChelseaRaisedNotificationModal notification={selectedNotification} />
+    )
+  }
+
+  if (
+    selectedNotification &&
+    selectedNotification.reason == "chelsea_st_bridge_lowered"
+  ) {
+    return <ChelseaLoweredNotificationModal />
+  }
```

Comment:
**issue:** sorry, one more thing that I should have caught in my original review, but `modal.test.tsx` should also have tests to make sure it renders the correct modals depending on the selected notification.